api/src/main/java/javax/enterprise/event/Event.java

```diff
@@ -100,6 +100,30 @@
 
     /**
      * <p>
+     * Fires an event with the specified qualifiers and notifies observers.
+     * </p>
+     *
+     * @param event the event object
+     * @throws IllegalArgumentException if the runtime type of the event object contains a type variable
+     * @throws FireAsyncException   if a notified observer throws a checked exception, it will be wrapped and rethrown as an
+     *                                  (unchecked) {@link FireAsyncException}
+     */
+    public void fireAsync(T event);
```

Comment:
the method signature does not match the one introduced in the specification document